src/main/java/com/redhat/devtools/intellij/common/utils/ExecHelper.java

```diff
@@ -356,39 +357,63 @@ public Process destroyForcibly() {
     public boolean isAlive() {
       return delegate.isAlive();
     }
+
+    @Override
+    public boolean isRunning() {
+      return delegate.isAlive();
+    }
+
+    @Override
+    public void setWinSize(WinSize winSize) {
+
+    }
+
+    @Override
+    public WinSize getWinSize() throws IOException {
+      return null;
+    }
+
+    @Override
+    public int getPid() {
+      return 0;
+    }
   }
   private static void executeWithTerminalInternal(Project project, String title, File workingDirectory,
                                                   boolean waitForProcessExit, Map<String, String> envs,
                                                   String... command) throws IOException {
     try {
-      ProcessBuilder builder = new ProcessBuilder(command).directory(workingDirectory).redirectErrorStream(true);
-      builder.environment().putAll(envs);
-      Process p = builder.start();
+      PtyProcessBuilder builder = new PtyProcessBuilder(command)
+              .setDirectory(workingDirectory.getAbsolutePath())
+              .setRedirectErrorStream(true);
+      Map<String, String> env = System.getenv();
+      env.putAll(envs);
```

Comment:
this doesn't work as `System.getenv()` returns an unmodifiable map and putAll fails.

Reply:
@lstocchi thx for spotting. Fixing.

Reply:
@lstocchi fixed by
`new HashMap<>(System.getenv())`